### ISSUE TITLE
dry run issue

### ISSUE THREAD

**Issue by wpoely86** [hpcugent/easybuild-framework#1457]
When installing `Python-2.7.10-gimkl-2.11.5.eb` with develop (today):
```
== 2015-11-06 14:49:53,310 runpy.EB_Python INFO Sanity check for nose successful!
== 2015-11-06 14:49:53,311 runpy.easyconfig.easyconfig WARNING Unable to resolve template value python -c "import %(ext_name)s" with dict {'nameletterlower': 'p', 'versionsuffix': '', 'toolchain_name': 'gimkl', 'namelower': 'python', 'version_minor': '7', 'name': 'Python', 'versionprefix': '', 'nameletter': 'P', 'version_major': '2', 'version_major_minor': '2.7', 'version': '2.7.10', 'toolchain_version': '2.11.5'}
== 2015-11-06 14:49:53,311 runpy.easyconfig.easyconfig WARNING Unable to resolve template value python -c "import %(ext_name)s" with dict {'nameletterlower': 'p', 'versionsuffix': '', 'toolchain_name': 'gimkl', 'namelower': 'python', 'version_minor': '7', 'name': 'Python', 'versionprefix': '', 'nameletter': 'P', 'version_major': '2', 'version_major_minor': '2.7', 'version': '2.7.10', 'toolchain_version': '2.11.5'}
== 2015-11-06 14:49:53,312 runpy.EB_Python DEBUG starting sanity check for extension with filter ('python -c "import %(ext_name)s"', '')
== 2015-11-06 14:49:53,313 runpy.EB_Python DEBUG modulename found in self.options, using it: numpy
== 2015-11-06 14:49:53,313 runpy.run DEBUG run_cmd: running cmd python -c "import numpy" (in /tmp/vsc40003/easybuild)
== 2015-11-06 14:50:01,256 runpy.run DEBUG cmd "python -c "import numpy"" exited with exitcode 0 and output:

== 2015-11-06 14:50:01,599 runpy ERROR Traceback (most recent call last):
  File "/vscmnt/gent_vulpix/_/user/home/gent/vsc400/vsc40003/easybuild_easyinstalled/lib/python2.6/site-packages/easybuild_framework-2.4.0.dev0-py2.6.egg/easybuild/main.py", line 112, in build_and_install_software
    (ec_res['success'], app_log, err) = build_and_install_one(ec, init_env)
  File "/user/home/gent/vsc400/vsc40003/easybuild_easyinstalled/lib/python2.6/site-packages/easybuild_framework-2.4.0.dev0-py2.6.egg/easybuild/framework/easyblock.py", line 2215, in build_and_install_one
    result = app.run_all_steps(run_test_cases=run_test_cases)
  File "/user/home/gent/vsc400/vsc40003/easybuild_easyinstalled/lib/python2.6/site-packages/easybuild_framework-2.4.0.dev0-py2.6.egg/easybuild/framework/easyblock.py", line 2132, in run_all_steps
    self.run_step(step_name, step_methods)
  File "/user/home/gent/vsc400/vsc40003/easybuild_easyinstalled/lib/python2.6/site-packages/easybuild_framework-2.4.0.dev0-py2.6.egg/easybuild/framework/easyblock.py", line 2011, in run_step
    step_method(self)()
  File "/user/home/gent/vsc400/vsc40003/easybuild_easyinstalled/lib/python2.6/site-packages/easybuild_easyblocks-2.4.0.dev0-py2.6.egg/easybuild/easyblocks/p/python.py", line 177, in sanity_check_step
    super(EB_Python, self).sanity_check_step(custom_paths=custom_paths, custom_commands=custom_commands)
  File "/user/home/gent/vsc400/vsc40003/easybuild_easyinstalled/lib/python2.6/site-packages/easybuild_framework-2.4.0.dev0-py2.6.egg/easybuild/framework/easyblock.py", line 1663, in sanity_check_step
    self._sanity_check_step(*args, **kwargs)
  File "/user/home/gent/vsc400/vsc40003/easybuild_easyinstalled/lib/python2.6/site-packages/easybuild_framework-2.4.0.dev0-py2.6.egg/easybuild/framework/easyblock.py", line 1806, in _sanity_check_step
    failed_exts = [ext.name for ext in self.ext_instances if not ext.sanity_check_step()]
  File "/user/home/gent/vsc400/vsc40003/easybuild_easyinstalled/lib/python2.6/site-packages/easybuild_easyblocks-2.4.0.dev0-py2.6.egg/easybuild/easyblocks/n/numpy.py", line 254, in sanity_check_step
    return super(EB_numpy, self).sanity_check_step(custom_paths=custom_paths, custom_commands=custom_commands)
  File "/user/home/gent/vsc400/vsc40003/easybuild_easyinstalled/lib/python2.6/site-packages/easybuild_easyblocks-2.4.0.dev0-py2.6.egg/easybuild/easyblocks/generic/pythonpackage.py", line 288, in sanity_check_step
    return super(PythonPackage, self).sanity_check_step(*args, **kwargs)
  File "/user/home/gent/vsc400/vsc40003/easybuild_easyinstalled/lib/python2.6/site-packages/easybuild_framework-2.4.0.dev0-py2.6.egg/easybuild/framework/extensioneasyblock.py", line 125, in sanity_check_step
    extension=self.is_extension)
  File "/user/home/gent/vsc400/vsc40003/easybuild_easyinstalled/lib/python2.6/site-packages/easybuild_framework-2.4.0.dev0-py2.6.egg/easybuild/framework/easyblock.py", line 1660, in sanity_check_step
    if self.dry_run:
AttributeError: 'EB_numpy' object has no attribute 'dry_run'
 (at easybuild/main.py:145 in build_and_install_software)
ERROR: Traceback (most recent call last):
  File "/vscmnt/gent_vulpix/_/user/home/gent/vsc400/vsc40003/easybuild_easyinstalled/lib/python2.6/site-packages/easybuild_framework-2.4.0.dev0-py2.6.egg/easybuild/main.py", line 112, in build_and_install_software
    (ec_res['success'], app_log, err) = build_and_install_one(ec, init_env)
  File "/user/home/gent/vsc400/vsc40003/easybuild_easyinstalled/lib/python2.6/site-packages/easybuild_framework-2.4.0.dev0-py2.6.egg/easybuild/framework/easyblock.py", line 2215, in build_and_install_one
    result = app.run_all_steps(run_test_cases=run_test_cases)
  File "/user/home/gent/vsc400/vsc40003/easybuild_easyinstalled/lib/python2.6/site-packages/easybuild_framework-2.4.0.dev0-py2.6.egg/easybuild/framework/easyblock.py", line 2132, in run_all_steps
    self.run_step(step_name, step_methods)
  File "/user/home/gent/vsc400/vsc40003/easybuild_easyinstalled/lib/python2.6/site-packages/easybuild_framework-2.4.0.dev0-py2.6.egg/easybuild/framework/easyblock.py", line 2011, in run_step
    step_method(self)()
  File "/user/home/gent/vsc400/vsc40003/easybuild_easyinstalled/lib/python2.6/site-packages/easybuild_easyblocks-2.4.0.dev0-py2.6.egg/easybuild/easyblocks/p/python.py", line 177, in sanity_check_step
    super(EB_Python, self).sanity_check_step(custom_paths=custom_paths, custom_commands=custom_commands)
  File "/user/home/gent/vsc400/vsc40003/easybuild_easyinstalled/lib/python2.6/site-packages/easybuild_framework-2.4.0.dev0-py2.6.egg/easybuild/framework/easyblock.py", line 1663, in sanity_check_step
    self._sanity_check_step(*args, **kwargs)
  File "/user/home/gent/vsc400/vsc40003/easybuild_easyinstalled/lib/python2.6/site-packages/easybuild_framework-2.4.0.dev0-py2.6.egg/easybuild/framework/easyblock.py", line 1806, in _sanity_check_step
    failed_exts = [ext.name for ext in self.ext_instances if not ext.sanity_check_step()]
  File "/user/home/gent/vsc400/vsc40003/easybuild_easyinstalled/lib/python2.6/site-packages/easybuild_easyblocks-2.4.0.dev0-py2.6.egg/easybuild/easyblocks/n/numpy.py", line 254, in sanity_check_step
    return super(EB_numpy, self).sanity_check_step(custom_paths=custom_paths, custom_commands=custom_commands)
  File "/user/home/gent/vsc400/vsc40003/easybuild_easyinstalled/lib/python2.6/site-packages/easybuild_easyblocks-2.4.0.dev0-py2.6.egg/easybuild/easyblocks/generic/pythonpackage.py", line 288, in sanity_check_step
    return super(PythonPackage, self).sanity_check_step(*args, **kwargs)
  File "/user/home/gent/vsc400/vsc40003/easybuild_easyinstalled/lib/python2.6/site-packages/easybuild_framework-2.4.0.dev0-py2.6.egg/easybuild/framework/extensioneasyblock.py", line 125, in sanity_check_step
    extension=self.is_extension)
  File "/user/home/gent/vsc400/vsc40003/easybuild_easyinstalled/lib/python2.6/site-packages/easybuild_framework-2.4.0.dev0-py2.6.egg/easybuild/framework/easyblock.py", line 1660, in sanity_check_step
    if self.dry_run:
AttributeError: 'EB_numpy' object has no attribute 'dry_run'
```

**Comment by wpoely86** [comment]
Probably fixed by hpcugent/easybuild-framework#1455

**Comment by boegel** [comment]
indeed, #1455 fixes this bug
